lib/irc/BridgedClient.js

```diff
@@ -142,7 +142,6 @@ BridgedClient.prototype.connect = Promise.coroutine(function*() {
 
         this.inst = connInst;
         this.unsafeClient = connInst.client;
-        this.log.debug("connected!");
```

Comment:
Well, it's only logged for `debug`. Why remove it exactly?

Reply:
We already log "Connecting" and if it were to fail, we'd see an error. Plus, it's not actually logging anything about *which* connection is connected so it's not much use on it's own.

Reply:
Fair enough!